.gitignore

```diff
@@ -0,0 +1,2 @@
+# ignore generated manifests
+features/performance/manifests/generated
```

Comment:
not sure what others think, but I'd prefer to have the generated manifests in git as well...

Reply:
I can add it with some values for `RESERVED_CPUS` and `ISOLATED_CPUS`, but we can not give a good prediction, so it will be just some dummy values, it the main reason why I removed it.

Reply:
I see, probably it's a good idea to not commit them then.